docs/quick-answers/websites/how-to-clear-cache-and-cookies-in-browser/index.md

```diff
@@ -0,0 +1,208 @@
+
+A private browser window can be opened in Firefox.
+
+- Private browsing allows you to surf the web without saving cookies or browsing history or from using any previously cached data.
+- It does not mean you can mask your identity on the web or your activity online.
+- Data is cleared upon exiting the browser window.
+
+1.  To open a private browsing session, click the **File** menu and select **New Private Window**.
+
+      - You can also open a private window with the keyboard shortcut **Ctrl** + **Shift** + **P** (Windows and Linux) or **Command** + **Shift** + **P** (MacOS).
+
+## Clear the Cache in Safari
+
+1. From the **Safari** menu, select **Clear History**.
+   ![Safari Clear History](cache-safari.png)
+2. Select the desired time range option in the **Clear** drop-down menu.
+
+      To clear all the cache select **all history**, and then click **Clear History**.
+
+3. Go to the **Safari** menu and click **Quit Safari** to exit the browser.
+
+      On MacOS, press **Command** + **Q** to exit the browser completely.
+
+### Disable Cache with Developer Tools
+
+Safari has a Developer tools dashboard built in, but you may need to enable it. Within this you have the ability to disable caching in a specific tab.
+
+1.  First, to enable it, click the **Safari** menu, **Preferences**. Then in the **Advanced** tab, choose **Show Develop menu in menu bar**.
+
+      ![Safari Preferences Show Develop Menu](safari-show-develop-menu.png "Safari Preferences Show Develop Menu")
+
+1.  Close the preference panel. The **Develop** menu should now appear on the menu bar.
+
+1.  To open the Developer tools panel, click on the **Develop** menu, then click **Show Web Inspector**.
+
+1.  The dashboard will open at the bottom of the screen.
+
+1.  Click the **Network** tab, and select the icon for **Ignore the resource cache when loading resources**.
+
+      ![Safari Web Developer Tools Disabled Cache](safari-disable-cache-developer-tools.png "Safari Web Developer Tools Disabled Cache")
+
+1.  Reload the page. This will load the page with the cache disabled.
+
+### Private Browsing in Safari
+
+A private browser window can be opened in Safari.
+
+- Private browsing allows you to surf the web without saving cookies or browsing history.
+- It does not mean you can mask your identity on the web or your activity online.
+- Additionally, tabs in a private browser window are isolated from each other and private browsing windows can not be passed to other Apple devices via Handoff.
+- Data is cleared upon exiting the browser window.
+
+1.  To open a private browsing session, click the **File** menu and select **New Private Window**.
+
+      - You can also open a private window with the keyboard shortcut **Ctrl** + **Shift** + **N** (Windows and Linux) or **Command** + **Shift** + **N** (MacOS).
+
+## Clear the Cache in Edge
+
+1. While in the browser press **Ctrl** + **Shift** + **Delete** keys. The **Clear browsing data** window appears.
```

Comment:
We should make this section look like the other keyboard shortcut sections, with a blockquote, just to be consistent.